source/common/config/grpc_mux_impl.h

```diff
@@ -42,7 +43,9 @@ class GrpcMuxImpl : public GrpcMux,
   void onRemoteClose(Grpc::Status::GrpcStatus status, const std::string& message) override;
 
   // TODO(htuch): Make this configurable or some static.
-  const uint32_t RETRY_DELAY_MS = 5000;
+  const uint32_t RETRY_INITIAL_DELAY_MS = 5000;
+  const uint32_t RETRY_MAX_DELAY_MS = 120000; // Do not cross more than 2 mins
```

Comment:
Seems too large IMHO for default, 2 minutes is a long time.. How about 30s?

Reply:
I am fine. Just put some number so that we can discuss